phpunit.xml

```diff
@@ -2,6 +2,7 @@
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:noNamespaceSchemaLocation="./vendor/phpunit/phpunit/phpunit.xsd"
          bootstrap="./vendor/autoload.php"
+         cacheResult="false"
```

Comment:
could you please explain why?

Reply:
The `.phpunit.cache` files was created, so it is this or adding that file to `.gitignore`. Since there are 4 tests here caching them for anything won't be any benefit.